### ISSUE TITLE
IDA 7.0

### ISSUE THREAD

**Issue by ApoIlos** [emoose/idaxex#2]
Could you post a 7.0 version. i tried to build it myself but got some errors that i can't fix,

**Comment by ReclaimerShawn** [comment]
Yes, this would be highly appreciated. I use 7.0 for reverse engineering other games, so I'd really like to not have to download another version of IDA just for the Xbox 360.
